app/channels/queue_channel.rb

```diff
@@ -138,6 +138,21 @@ def destroy_request(data)
     broadcast_request_change('resolve_request', request)
   end
 
+  def merge(data)
+    authorize :instructor_only
+    @to_course_queue = CourseQueue.find_by!(id: data['to'])
+
+    @course_queue.outstanding_requests.each do |request|
+      request.update! course_queue_id: data['to']
+      broadcast_request_change('resolve_request', request)
+      QueueChannel.broadcast_to(@to_course_queue, {
+        action: 'new_request',
+        request: serialize_request(request),
+      })
+    end
```

Comment:
What does the UX for the student look like? How do they know their request has moved to another queue?

Reply:
Now, the user receives an alert of which queue they were moved to. Once the alert is acknowledged, the user is forwarded to the queue they were moved to.